app/services/code_hosting/git_hub_service.rb

```diff
@@ -69,6 +69,11 @@ def username
       client.login
     end
 
+    # Primary email address of the current GitHub user
+    def email
+      client.emails.find { |e| e[:primary] == true }[:email]
```

Comment:
we use this shorter form in other places, any reason why we shouldn't use it here?: `client.emails.find(&:primary).email`

Reply:
So fancy, so much magic, will update 👍 I just never think about it